styles/widgets/common/draggable.less

```diff
@@ -13,5 +13,5 @@
     position: absolute;
     top: 0;
     left: 0;
-    z-index: 10000;
+    z-index: 2147483647;
```

Comment:
Should it be like an inline number? Constant is better for it (MAX_Z_INDEX e.g.)